test/e2e/util/util.go

```diff
@@ -847,7 +847,7 @@ func EnsureNetworkPolicies(t *testing.T, ctx context.Context, c crclient.Client,
 			}
 
 			// Validate cluster-version-operator is not allowed to access management KAS.
-			_, err = RunCommandInPod(ctx, c, "cluster-version-operator", hcpNamespace, command, "cluster-version-operator")
+			_, err = RunCommandInPod(ctx, c, "cluster-version-operator", hcpNamespace, command, "cluster-version-operator", 0)
```

Comment:
Not super thrilled about how `RunCommandInPod` effectively needs to grow in order to expose all of the pod execution options that are otherwise so nicely captured in the executor struct, but whatever